src/test/FuzzerImpl.cpp

```diff
@@ -602,6 +606,11 @@ TransactionFuzzer::initialize()
     // We don't issue assets from account 0.
     accountParameters[0].mNumAssetsToDistribute = 0;
 
+    // Make some issuing accounts require authorization.
+    accountParameters[1].mAccountFlags = AUTH_REQUIRED_FLAG;
+    accountParameters[2].mAccountFlags =
+        AUTH_REQUIRED_FLAG | AUTH_REVOCABLE_FLAG;
```

Comment:
You also probably want some accounts with `AUTH_IMMUTABLE_FLAG`.

Reply:
I think this belongs to a different PR: I think this should be covered by
`Trustlines and orders seem "uninteresting". Setup is fully symetrical. No variations on issuer/trustline flags.`

Reply:
> I think this belongs to a different PR: I think this should be covered by
Trustlines and orders seem "uninteresting". Setup is fully symetrical. No variations on issuer/trustline flags.

Yeah, that's what I was working on there.  I've removed that commit from this PR (and put it in a new branch).